web/elm/src/Pipeline.elm

```diff
@@ -234,6 +234,10 @@ view model =
             , Html.dd [] [ Html.text "aborted" ]
             , Html.dt [ class "paused" ] []
             , Html.dd [] [ Html.text "paused" ]
+            , Html.dt [ class "dotted" ] [ Html.text "." ]
+            , Html.dd [] [ Html.text "Manually triggered job" ]
```

Comment:
I suggest you respect the case of the elements above, this `, Html.dd [] [ Html.text "manually triggered job" ]`. Same goes for line `240`.

As for the language, I'm not sure `manually trigerred job` is appropriate for the following reasons :
- [One can disable manual triggers](https://concourse.ci/configuring-jobs.html#disable_manual_trigger), making this a misnomer.
- We're really talking about resources here.

I would suggest `dependency link` and `trigger dependency`... but I'm not even particularly excited about those suggestions.